### PR TITLE
fix: Missing placeholder configurer class.

### LINKED ISSUE #18: Missing Spring class
Cannot  find class  org.springframework.context.support.PortalPropertySourcesPlaceholderConfigurer

### PR DESCRIPTION
**Pull request by bjagg**
Fixes #18

### CODE REVIEW

**Comment by apetro** [comment]
+0. How might we architect this so that JSR portlets in uPortal don't need to depend on uPortal-specific libraries. This is just a matter of ensuring a shared `.properties` file is available to do properties placeholder configuration in `BookmarksPortlet`? There some less invasive way to do this?

File: pom.xml [hunk 1]
```diff
@@ -31,6 +31,7 @@
 
     <properties>
         <spring.version>3.2.13.RELEASE</spring.version>
+        <uportal-libs.version>5.0.4</uportal-libs.version>
```
Comment (ChristianMurphy) [inline pom.xml]: Could this be uPortal latest (5.0.5)?

**Comment by drewwills** [comment]
@apetro -- The needed class is an extension of Spring's `PropertySourcesPlaceholderConfigurer` that adds support for encrypted properties.  It doesn't use any "portal-tech" classes or resources.

We could make this arrangement less "messy" by spinning the class off into a different jar built from another repo.  I'm open to that.

**Comment by drewwills** [comment]
I'm thinking probably not, because Spring 5 already does it (iirc).

**Comment by apetro** [comment]
It might be that Spring 5 already supports encrypted properties, but I'm not seeing it in [the docs](https://docs.spring.io/spring/docs/current/spring-framework-reference/core.html#placeholder-resolution-in-statements).

**Pull request merged**